routes/api/accounts.js

```diff
@@ -0,0 +1,13 @@
+const express = require( 'express' )
+const router = express.Router()
+
+const { register } = require( '../accounts/accounts_route' )
+
+router.post('/register', ( request, response ) => {
+  register(
+    request,
+    response.status( 202 ).json( { message: 'Please check your email' }
```

Comment:
This code will always execute, meaning that a user would always receive the 202 message...